sktime/forecasting/compose/_fallback.py

```diff
@@ -36,6 +47,13 @@ class is useful in scenarios where the reliability of individual forecasting mod
     verbose : bool, default=False
         If True, raises warnings when a forecaster fails to fit or predict.
 
+    nan_predict_policy: str, default='ignore'
+        Determines the action to take if NaN values are found in the predictions. If set
+         to 'raise', the method raises a ValueError when NaN values are detected in the
```

Comment:
I would explain `raise` better - the exception is in fact not raised to hte user, but handled by this class, i.e., occurrence of nans is considered a failure for the fallback